pkg/nuctl/test/function_test.go

```diff
@@ -1536,6 +1536,44 @@ func (suite *functionExportImportTestSuite) TestAutofixWhenImportFunction() {
 	suite.Require().NotNil(err)
 }
 
+func (suite *functionExportImportTestSuite) TestImportWithReport() {
+	functionConfigPath := path.Join(suite.GetImportsDir(), "project_with_wrong_conf.yaml")
+	projectName := "test-project"
+	functionNames := []string{"incorrect-fixable-test-function", "incorrect-not-fixable-test-function", "correct-test-function"}
+	defer func() {
+		// delete project
+		suite.ExecuteNuctl([]string{"delete", "project", projectName}, nil) // nolint: errcheck
+		// delete functions
+		for _, funcName := range functionNames {
+			suite.ExecuteNuctl([]string{"delete", "fu", funcName}, nil) // nolint: errcheck
+		}
+	}()
```

Comment:
Does this mean that deleting a project does not delete its functions?? 👀 
What if you pass `strategy=cascading`?

Reply:
@TomerShor didn't know we have it, added cascading then